packages/shared/styles/components/molecules/SfBanner.scss

```diff
@@ -2,80 +2,82 @@
 
 .sf-banner {
   box-sizing: border-box;
-  display: flex;
-  min-height: var(--banner-min-height, 9rem);
-  padding: var(--banner-padding, var(--spacer-big));
-  color: var(--banner-color, var(--c-text));
-  background-repeat: no-repeat;
-  background-position: var(--banner-background-position, bottom left);
-  background-size: cover;
-  background-color: var(--banner-background);
+  display: var(--banner-display, flex);
+  flex-direction: var(--banner-flex-direction, column);
+  justify-content: var(--banner-justify-content, left);
+  background-color: var(--background-color);
+  background-position: var(--banner-background-position, center);
+  background-repeat: var(--banner-background-repeat, no-repeat);
+  background-size: var(--banner-background-size, cover);
   background-image: var(--background-image);
+  color: var(--banner-color, var(--c-text));
+  width: var(--banner-size, var(--banner-width, 100%));
+  min-height: var(--banner-size, var(--banner-min-height, 14rem));
+  letter-spacing: var(--banner-letter-spacing, 0.1px);
   &__container {
-    flex: var(--banner-container-flex, 0 0 100%);
+    display: flex;
+    justify-content: left;
+    flex-direction: column;
+    margin: var(--banner-container-margin, 3.125rem 0 0 3.125rem);
   }
   &__subtitle {
-    margin: var(--banner-subtitle-margin);
-    color: var(--banner-subtitle-color, var(--c-dark-variant));
-    font: var(
+    @include font(
```

Comment:
try to stick with ordering properties like here :) -> https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/